notebooks/book2/15/logreg_iris_bayes_2d.ipynb

```diff
@@ -1,49 +1,229 @@
 {
```

Comment:
<p>I think we should keep a comment pointing to the original pymc code at</p><p> </p><p>https://github.com/aloctavodia/BAP/blob/master/code/Chp4/04_Generalizing_linear_models.ipynb</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/994/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Ohh I missed it, Done.</p>